responses/en/HassPersonGetLocation.yaml

```diff
@@ -0,0 +1,5 @@
+language: en
+responses:
+  intents:
+    HassPersonGetLocation:
+      location: "{slots.name} is in the {slots.state}"
```

Comment:
This response needs more thought. It's works grammatically for your room examples, but not for zones which I would assume is the more common use case. 

- is in the away
- is in the home
- is in the Dads house
- is in the Sydney
- is in the dancing

Perhaps attributes of the state object could be used in a more complex template to distinguish between cases? 
- is away
- is at Dad's house
- is in the kitchen
- is driving in (suburb) ??

Or a generic response
- (person)s location is state.state

Reply:
Sure - this was intended as an MVP to try feel out intents and how they work.

It seems that this isn't quite the right place/time for that yet though.